src/plugins/dam/sagas/patchAssetsFlow.js

```diff
@@ -45,13 +45,17 @@ export default function* (action) {
   const expectedEvent = config.schemas.assetPatched.getCurie().toString();
 
   try {
+    let timeout = 5000;
+    if (config.assetCount && config.assetCount > 10) {
+      timeout += (500 * (config.assetCount - 10));
```

Comment:
@hunyoboy what does this accomplish?

Reply:
@efjacobson i'm glad you mentioned. so this is based from my past update here - https://github.com/triniti/cms-js/blob/master/src/plugins/dam/sagas/reorderGalleryAssetsFlow.js#L90 w/c is @gdbrown 's idea. i'm thinking this is applicable here since both actions  are using `2pc`.  what do you think Greg?